sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/AbstractLNAdapter.java

```diff
@@ -159,7 +159,9 @@ public String getLnType() {
      * @return list of LNode ExtRefs elements
      */
     public List<TExtRef> getExtRefs() {
-        return getExtRefs(null);
+        List<TExtRef> tExtRefs =  getExtRefs(null);
+        if(tExtRefs == null || tExtRefs.isEmpty()) return Collections.emptyList();
+        else return tExtRefs;
```

Comment:
It returned null.

Reply:
Do you have a test where it returns null ?
Because I don't see how `getExtRefs(ExtRefSignalInfo filter)` can return null. It returns either : 

- `return new ArrayList<>();`
- `return currentElem.getInputs().getExtRef();` => never null if you look at `TInputs.getExtRef()`implementation
- `return currentElem.getInputs().getExtRef().stream().filter(...).toList();`

and here is the code for `TInputs.getExtRef()`
```java
public class TInputs extends TUnNaming {
...
    public List<TExtRef> getExtRef() {
        if (extRef == null) {
            extRef = new ArrayList<TExtRef>();
        }
        return this.extRef;
    }
```

Reply:
right!